time stamp
float32 velocity
float32 omega 
